--- libchamplain-0.3.3/champlain/Makefile.am~	2009-06-15 04:52:12.000000000 +0200
+++ libchamplain-0.3.3/champlain/Makefile.am	2009-06-16 19:39:00.186972692 +0200
@@ -67,7 +67,6 @@
 	champlain-network-map-source.h	\
 	champlain-map-source-factory.h	\
 	champlain-map-source-desc.h	\
-	champlain-version.h		\
 	champlain-cache.h		\
 	champlain-polygon.h		\
 	champlain-version.h
@@ -86,7 +86,6 @@
 	champlain-zoom-level.h		\
 	champlain-base-marker.h		\
 	champlain-marker.h		\
-	champlain-version.h		\
 	champlain-cache.h		\
 	champlain-polygon.h		\
 	champlain-version.h
